<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Morphing</class>
 <widget class="QWidget" name="Morphing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame_left">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_right">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>0</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QSlider" name="keyframe">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>550</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="keyframe_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>520</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Keyframe</string>
   </property>
  </widget>
  <widget class="QLabel" name="animation_label">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>520</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Animation</string>
   </property>
  </widget>
  <widget class="QSlider" name="animation">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>550</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="frames_count_label">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>520</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Frames count</string>
   </property>
  </widget>
  <widget class="QSlider" name="frames_count">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>550</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>50</number>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="value">
    <number>275</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="play">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>540</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Play animation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>540</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
